project/dexter-core/src/java/com/samsung/sec/dexter/core/config/PlatzKeywordFile.java

```diff
@@ -0,0 +1,59 @@
+package com.samsung.sec.dexter.core.config;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Map;
+import com.google.common.base.Charsets;
+import com.google.common.io.Files;
+import com.google.gson.Gson;
+import com.samsung.sec.dexter.core.exception.DexterRuntimeException;
+
+public class PlatzKeywordFile {
+	private String keyword;
+	
+	public void loadFromFile(final File file){
+		Map<String, Object> keywordMap = getKeywordMap(file);
+		setFields(keywordMap);
+		
+	}
+	
+	protected Map<String, Object> getKeywordMap(final File keywordFile) {
+		if (keywordFile.exists() == false) {
+			throw new DexterRuntimeException("There is no " + DexterConfig.PLATZ_KEYWORD_FILENAME 
+					+ " file : " + keywordFile.getName());
+		}
+		final StringBuilder keywordJson = new StringBuilder(50);
+		try {
+			for (String content : Files.readLines(keywordFile, Charsets.UTF_8)) {
+				keywordJson.append(content);
+				if (content.indexOf("}") >= 0) {
+					break;
+				}
+			}
+			final Gson gson = new Gson();
+            @SuppressWarnings("unchecked")
+            Map<String, Object> keywordMap = gson.fromJson(keywordJson.toString(), Map.class);
+            
+            
+            return keywordMap;
```

Comment:
Please, recheck the final handler for these Exceptions.
If there is no handler, the monitoring thread can be stopped.